spec/car_spec.rb

```diff
@@ -37,4 +40,12 @@
 
     expect(car.total_cost).to eq(54000)
   end
+
+  it "has color" do
+    expect(car.color).to eq([])
+  end
+
+  it "is a color of paint" do
+  expect(car.paint!).to add(:blue)
+  end
```

Comment:
We want to pass a parameter to `.paint!`

```suggestion
  it "is a color of paint" do
    car.paint!("blue")
    expect(car.color).to eq("blue")
  end
```